samples/server/pom.xml

```diff
@@ -128,8 +127,52 @@
           </execution>
         </executions>
       </plugin>
-
     </plugins>
   </build>
-
-</project>
+  <dependencyManagement>

```

Comment:
I think we should address this also differently... too many exclusions

Maybe we should exclude spring-boot-stater-tomcat from the base archetype and sample and only add it when using a "boot" or "standalone" profile. (that would be a profile added by default)

then if you have to deploy on other server you simply remove this profile and you don't have to remove anything.

Also, you can have exclusions near where they are being included as seen here (I've just found it)
https://spring.io/blog/2014/03/07/deploying-spring-boot-applications

So you give information where the dependency comes in the first place.